src/PantherTestCaseTrait.php

```diff
@@ -162,10 +161,22 @@ protected static function createPantherClient(array $options = [], array $kernel
 
         self::startWebServer($options);
 
-        if (PantherTestCase::CHROME === $browser) {
-            self::$pantherClients[0] = self::$pantherClient = Client::createChromeClient(null, null, $managerOptions, self::$baseUri);
-        } else {
+        if (PantherTestCase::FIREFOX === $browser) {
             self::$pantherClients[0] = self::$pantherClient = Client::createFirefoxClient(null, null, $managerOptions, self::$baseUri);
+        } else {
+            try {
+                self::$pantherClients[0] = self::$pantherClient = Client::createChromeClient(null, null, $managerOptions, self::$baseUri);
+            } catch (\RuntimeException $e) {
+                if (PantherTestCase::CHROME === $browser) {
+                    throw $e;
+                }
+                $managerOptions['capabilities']['acceptInsecureCerts'] ??= true;
```

Comment:
I'm not sure this is the proper location for this. Should we could move it to the `Client` class?

I added this line because while the "Symfony dev CA" cert authority is installed for the regular firefox profile, it's not in headless mode, and I don't know why nor if it's possible to change that.

(I didn't check this aspect with Chrome yet.)

Reply:
Now tested on Chrome, which works like a charm.